erica/elster_xml/common/elsterify_fields.py

```diff
@@ -13,6 +14,8 @@ def elsterify_anrede(anrede_input: Anrede):
     return anrede_mapping[anrede_input]
 
 
-def elsterify_date(date_input: date):
+def elsterify_date(date_input: Union[date, None]):
     """ Converts input date to Elster's date format """
+    if not date_input:
```

Comment:
As we had this lead to problems a few times, I personally prefer to test explicitly for None if that is what we want to do.

It's fine here though